src/components/Shared/Modal.js

```diff
@@ -92,7 +93,7 @@ function SharedModal(props) {
     closeModal();
   };
 
-  return (
+  return ReactDom.createPortal(
```

Comment:
```suggestion
  return createPortal(
```
여기는 이렇게... 어떻게 쓸 지는 정하기 나름이니까 한번 얘기해봐요!!